proxy/coco/src/state/error.rs

```diff
@@ -28,6 +28,10 @@ pub enum Error {
     #[error(transparent)]
     Create(#[from] crate::project::create::Error),
 
+    /// An error occurred while validating the path for creating a project.
+    #[error(transparent)]
+    CreateValidation(#[from] crate::project::create::validation::Error),
```

Comment:
So now we have `Error::Create` and `CreateValidation`. I was expecting the creation to validate, i.e, that these were blent. Can you help me understand?

Reply:
The idea was to separate the validation and creation. But you have a point it should happen at both points really.